.gitignore

```diff
@@ -13,6 +13,8 @@ target
 scratch
 .vscode
 
+package-lock.json
```

Comment:
probably should add yarn.lock as well if we are going to start ignoring lock files